kubernetes/__init__.py

```diff
@@ -16,10 +16,11 @@
 # The version is auto-updated. Please do not edit.
```

Comment:
this file is auto generated. Please fix it in https://github.com/OpenAPITools/openapi-generator

Reply:
I made a simple report here https://github.com/OpenAPITools/openapi-generator/issues/17152